changelog.d/20251217_150504_athornton_DM_53638_simplectrs.md

```diff
@@ -0,0 +1,15 @@
+<!-- Delete the sections that don't apply -->
+
+### Backwards-incompatible changes
+
+- `config.lab.tmpSource` renamed to `config.lab.emptyDirSource`.
```

Comment:
I would also add a bit more description like:
 This setting now controls the source for both the `/tmp` emptyDir volume and the new `/lab_startup` emptyDir volume.